api/FileReaderSync.json

```diff
@@ -119,7 +119,7 @@
               "version_added": "8"
             },
             "ie": {
-              "version_added": true
+              "version_added": "11"
```

Comment:
Curious that `readAsBinaryString` would be added after `readAsArrayBuffer`, but I see no reason to doubt the results.